components/button/button--header.hbs

```diff
@@ -1,3 +1,3 @@
-<div class="bar u-pl-m u-pt-m">
+<div class="bar u-pl-m u-pt-m u-pb-m">
```

Comment:
Potential overuse of utility classes, should be used sparingly. Prefer adding these single properties into the block, or create a modifier if a new variant is needed.

Reply:
As discussed, these classes are only added to display the button and isn't part of the code that would be used.